src/qt/guiutil.cpp

```diff
@@ -749,6 +749,14 @@ QString boostPathToQString(const fs::path &path)
     return QString::fromStdString(path.string());
 }
 
+QString NetworkNameToQString(const std::string& network)
+{
+    if (network == "ipv4") return "IPv4";
+    if (network == "ipv6") return "IPv6";
+    if (network == "onion") return "Onion";
+    return "";
+}
+
```

Comment:
97d5b82beaf35761bfcc31562f525f4bc52d6529

It seems strange for me to add a new function for the sake of a such functionality. And string interface is fragile.

Why do not expose `Network` in `CNodeStats` (the same approach you use in #163)?

Reply:
Agreed, I wanted to both here and in #163 but I didn't think it would be accepted by reviewers. Updating to switch on the enum instead and add i2p and cjdns.

Reply:
done -- I agree it's a shame to add a function but the GUI UI is non all-lowercase like the RPC/CLI interface